guides/common/modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc

```diff
@@ -1,11 +1,11 @@
 [id="enabling-and-disabling-repositories-on-activation-key_{context}"]
 = Enabling and Disabling Repositories on Activation Key
 
-Use this procedure to enable or disable repositories on an activation key.
+As a Simple Content Access (SCA) user, you can enable all custom repositories on an activation key in the {ProjectWebUI}.
```

Comment:
Using this narrow overview sentence but keeping the broader instructions again means these don't match. 
```suggestion
As a Simple Content Access (SCA) user, you can enable or disable repositories on an activation key in the {ProjectWebUI}.
```